internal/watcher/watcher_test.go

```diff
@@ -162,8 +162,12 @@ func TestWatchDirectory(t *testing.T) {
 						require.NoError(t, err, "Can't read file")
 						d = append(data, []byte("\n;comment string")...)
 					}
-					err := os.WriteFile(filepath.Join(temp, path), d, 0600)
-					require.NoError(t, err, "Can't update file")
+					f, err := os.OpenFile(filepath.Join(temp, path), os.O_CREATE|os.O_RDWR, os.ModePerm)
```

Comment:
Yeah, as Gabriel told, definitively extracting this in testutils as we are using it 4 times, with proper documentation, blablabla.